lang/src/org/partiql/lang/planner/transforms/LogicalToLogicalResolvedVisitorTransform.kt

```diff
@@ -392,12 +392,13 @@ internal data class LogicalToLogicalResolvedVisitorTransform(
 
     override fun transformBexprAggregate(node: PartiqlLogical.Bexpr.Aggregate): PartiqlLogicalResolved.Bexpr {
         val scope = getOutputScope(node.source).concatenate(this.inputScope)
+        val keysScope = node.groupList.keys.getOutputScope()
         return PartiqlLogicalResolved.build {
             aggregate(
                 source = transformBexpr(node.source),
                 strategy = super.transformBexprAggregate_strategy(node),
                 groupList = withInputScope(scope) { super.transformBexprAggregate_groupList(node) },
-                functionList = withInputScope(scope) { super.transformBexprAggregate_functionList(node) },
+                functionList = withInputScope(scope.concatenate(keysScope)) { super.transformBexprAggregate_functionList(node) },
```

Comment:
I don't think you would need the .super call here. 

Also, discuss the `scope.concatenate(keyScope)` offline.